content/docs/install/linux.md

```diff
@@ -29,7 +29,7 @@ plan to use, you might need to install optional dependencies: `[s3]`, `[azure]`,
 ### Example: How to install DVC with support for Amazon S3 storage
 
 ```dvc
-$ pip install 'dvc[s3]'
+$ pip install "dvc[s3]"
```

Comment:
they didn't work in some environment? did you check fish, zsh, bash?

Reply:
most likely you are right and we need double quotes ... just to make sure

Reply:
Yes I remember it doesn't work with single quote in some env, Windows I think. We have notes about installing these optional reqs in other docs and they use `"` for that reason.

Reply:
Seems to work on Windows cmd. Sorry, I don't remember what env but its out there. I'll ask @efiop, he probably remembers.

Reply:
He thought also Windows. Maybe they fixed it? `"` works everywhere anyway so let's keep?

Reply:
👍